app/mapping/UserAnswersExtractor.scala

```diff
@@ -43,15 +43,19 @@ class UserAnswersExtractorImpl @Inject()(
 
   override def extract(answers: UserAnswers, data: GetTrust): Either[PlaybackExtractionError, UserAnswers] = {
 
+    val isTrustTaxable: Boolean = !data.trust.details.trustTaxable.contains(false)
```

Comment:
Thinking ahead it might be a nice bit of refactoring to add `def isTaxable = !trustTaxable.contains(false)` to the `TrustDetailsType` case class then each extractor/mapper can use that